src/StoreKeeper/WooCommerce/B2C/Imports/ProductUpdateImport.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Imports;
+
+use Adbar\Dot;
+use DateTime;
+use Exception;
+use StoreKeeper\WooCommerce\B2C\Exceptions\WordpressException;
+use StoreKeeper\WooCommerce\B2C\I18N;
+use WC_Data_Exception;
+
+class ProductUpdateImport extends ProductImport
+{
+    /**
+     * @throws WC_Data_Exception
+     * @throws WordpressException
+     * @throws Exception
+     */
+    protected function processSimpleAndConfigurableProduct(
```

Comment:
you should update the fields based on the event and details. Implemented that way won't update images for example.

* `event = updated && details = 'product_price'` -> product price
* `event = updated && details = 'product.product_stock'` -> stock change
* `event = updated && details = 'product.upsell_products'` -> upsell product change
* `event = updated && details = 'product.cross_sell_products'` -> cross_sell product change
* `other cases` -> full product

Reply:
Added `scopes` for isolating product updates.